tools/changelog-generator/README.md

```diff
@@ -27,14 +27,17 @@ To generate a `CHANGELOG.md` file for all released Kyma versions, follow these s
 1. Create a `CHANGELOG.md` file under the specified absolute path to the repository. Use this command:
 
 ```bash
-docker run --rm -v {absolutePathToRepository}:/repository -w /repository -e LATEST_VERSION={applicationVersion} -e GITHUB_AUTH={githubToken} -e SSH_FILE={sshFilePath} changelog-generator sh /app/generate-full-changelog.sh --configure-git
+docker run --rm -v {absolutePathToRepository}:/repository -w /repository -e LATEST_VERSION={applicationVersion} -e GITHUB_AUTH={githubToken} -e SSH_FILE={sshFilePath} -e CONFIG_FILE={configFilePath} -e SKIP_REMOVING_LATEST=true changelog-generator sh /app/generate-full-changelog.sh --configure-git
```

Comment:
Please rename `LATEST_VERSION` to something like `NEW_RELEASE_TITLE` or something like that - as it is only used for title in changelog, right?

Reply:
It is used to changelog file tittle, in commit message (during pull changelog file to root repository) and if "FROM_TAG" is used then learna-changelog got it as a "to" argument. I changed the name.

Reply:
> if "FROM_TAG" is used then learna-changelog got it as a "to" argument

Hmm, but to be honest it should be used in this way, as by design the tag shouldn't exist yet 😛